lib/mix/tasks/conduit.gen.broker.ex

```diff
@@ -204,6 +204,21 @@ defmodule Mix.Tasks.Conduit.Gen.Broker do
   <% end %>
   Also, add your broker to the supervision hierarchy in your <%= @app %>.ex:
 
+  Elixir v1.5 or above:
+
+      def start(_type, _args) do
+        children = [
+          # ...
+          {<%= @module %>, []}
+        ]
+
+        opts = [strategy: :one_for_one]
+
+        Supervisor.start_link(children, opts)
```

Comment:
Should be using https://hexdocs.pm/elixir/Supervisor.html#init/2

Reply:
You're right. In supervisor init's you use `Supervisor.init/2`, because the it expects a spec and the behavior starts them for you. I haven't had to modify an application start in a while, which tends to be very similar, so I assumed it needed a spec too. Good catch!